src/Symfony/Component/Notifier/Message/SmsMessage.php

```diff
@@ -93,6 +99,25 @@ public function getTransport(): ?string
         return $this->transport;
     }
 
+    /**
+     * @return $this
+     */
+    public function from(?string $from): static
```

Comment:
Not know how to deal with setter (Need change in MessageInterface and introduce BC Break).

(Also not sure if setter is really needed)